frontend/Rhyolite/Frontend/Form.hs

```diff
@@ -195,57 +209,142 @@ data ValidationInput t m e a = ValidationInput
   , _validationInput_value :: DynValidation t e a
   }
 
+data ValidationTextArea t m e a = ValidationTextArea
+  { _validationTextArea_input :: TextAreaElement EventResult (DomBuilderSpace m) t
+  , _validationTextArea_value :: DynValidation t e a
+  }
+
+data ValidationDropdown t e a = ValidationDropdown
+  { _validationDropdown_input :: Dropdown t (Maybe a)
+  , _validationDropdown_value :: DynValidation t e a
+  }
+
```

Comment:
Why do we need validation for a dropdown? Shouldn't we just only offer options that are valid?

Reply:
@eskimor One of the items in the dropbox can be "Select an item" (which would be the default selection). If the user has not in fact selected one of the _other_ items, we would want to display a validation error. 

The "Select an item" thing would be `Nothing`, and the other items `Just a`. So we would just use the `validateJust` function here.

Reply:
Makes sense. Thanks!